runtimes/python/catala/src/catala/runtime.py

```diff
@@ -70,6 +70,9 @@ def __eq__(self, other: object) -> bool:
 
```

Comment:
Good idea to put the runtimes in a separate folder. I should do the same for OCaml.